mindbender/maya/plugins/validate_project_editorial_info.py

```diff
@@ -0,0 +1,50 @@
+# validate_project_editorial_info.py
+import pyblish.api
+
+
+class ValidateMindbenderProjectEditInfo(pyblish.api.ContextPlugin):
+    """
+    All the info that gets validated
+    has been set by the projects
+    bat files.
+    If you are surtain that the info
+    is incorrect, talk to
+    project Supervisor with haste !
+
+    """
+
+    label = "Validate Project Edit Info"
+    optional = True
+    order = pyblish.api.ValidatorOrder
+    hosts = ["maya"]
+
+    def process(self, context):
+        from maya import cmds
+
+        sceneIn = cmds.playbackOptions(query=True, animationStartTime=True)
+        sceneOut = cmds.playbackOptions(query=True, animationEndTime=True)
```

Comment:
That is why I chose _aimationStartTime_.

I didn't clear any of this with the @CalleHalldin yet but if he wants then he can drop the mic on this one.

I thought it better to take animationStartTime because that is the one that we set in the [_usersetup.py_](https://github.com/mindbender-studio/core/blob/master/mindbender/maya/pythonpath/userSetup.py#L37-L43) 
So to me that was the natural choise to validate as I can see the animator using the playback for their own work and then forgetting to set playback to the edits **In** and **Out**
but the animation start and end should always be set to the shots info.

The biggest reason for this validator is too correct if the animator has by mistake changed the **AnimationStartTime** and **AnimationEndTime**